models/migrations/v93.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import "github.com/go-xorm/xorm"
+
+func addEmailNotificationEnabledToUser(x *xorm.Engine) error {
+	// Issue see models/user.go
+	type User struct {
+		EmailNotificationsPreference string `xorm:"NOT NULL DEFAULT 'enabled'"`
```

Comment:
default string is `VARCHAR(255)`, I think it's unnecessary to use 255 bytes. Maybe 10 or 20 is enough.